web_infrastructure/taiga_issue.py

```diff
@@ -87,7 +87,7 @@
     choices: ["present", "absent"]
     default: present
 author: Alejandro Guirao (@lekum)
-requirements: [python-taiga]
+requirements: [python-taiga(U(https://https://pypi.python.org/pypi/python-taiga/)) > 0.8.6]
```

Comment:
The Shippable job fails here because it's not YAML compliant; try with:
```yaml
requirements: ['python-taiga(U(https://https://pypi.python.org/pypi/python-taiga/)) > 0.8.6']
```

Or:
```yaml
requirements: ['python-taiga > 0.8.6']
```

Don't know if it'll do what you want, but at least it'll pass.